### DISCUSSION TITLE
dfu-tool missing convert

### DISCUSSION THREAD

**Discussion by iotauto** [fwupd/fwupd#3647] [Q&A]
Version 1.3 has a convert option to convert binary format to dfu

On version 1.5 this option is gone

**Comment by superm1** [comment]
IIRC this moved to `fwupdtool firmware-convert` didn't it?

**Comment by iotauto** [comment]
Thanks for the response , I will try it
________________________________

Greg
 
James
Engineering Manager
 
M: +61 (0) 409566119
E: ***@***.***
W: iotautomation.com.au
